gammapy/background/energy_offset_array.py

```diff
@@ -104,3 +127,73 @@ def plot_image(self, ax=None, offset=None, energy=None, **kwargs):
         image = ax.imshow(self.data, extent=extent, **kwargs)
         plt.colorbar(image)
         return ax
+
+    @classmethod
+    def read(cls, filename):
+        """Create `EnergyOffsetArray` from  fits file.
+
+        Parameters
+        ----------
+        filename : str
+            File name
+        """
+        hdu_list = fits.open(filename)
+        return cls.from_fits(hdu_list)
+
+    @classmethod
+    def from_fits(cls, hdu):
+        """Read `EnergyOffsetArray` from a fits binary table.
+
+        Parameters
+        ----------
+        hdu : `~astropy.io.fits.BinTableHDU`
+            HDU binary table for the EnergyOffsetArray.
+
+        Returns
+        -------
+        cube : `~gammapy.background.EnergyOffsetArray`
+            EnergyOffsetArray object.
+        """
+
+        header = hdu[1].header
+        data = hdu[1].data
+        # get offset and energy binning
+        offset_edges = _make_bin_edges_array(data['THETA_LO'].squeeze(), data['THETA_HI'].squeeze())
+        energy_edges = _make_bin_edges_array(data['ENERG_LO'].squeeze(), data['ENERG_HI'].squeeze())
```

Comment:
In the constructor I define the offset and the energy as Quantity with fixed units but maybe this is not a good idea since here when I read the offset and energy from the fits file I can't define Quantity object with the units in the header?